src/activity_item_handling.cpp

```
@@ -3818,111 +3821,173 @@ static std::optional<tripoint_bub_ms> find_refuel_spot_trap(
+    } else {
+        return 2;
+    }
+}
+
+static time_duration get_comestible_time_left( const item_location &loc )
+{
+    time_duration time_left = 0_turns;
+    const time_duration shelf_life = loc->is_comestible() ? loc->get_comestible()->spoils :
+                                     calendar::INDEFINITELY_LONG_DURATION;
+    if( shelf_life > 0_turns ) {
+        const item &it = *loc;
```

Clang (14.0.6)

[error] variable name 'it' is too short, expected at least 3 characters

(readability-identifier-length,-warnings-as-errors)